akka-actor-testkit-typed/src/main/scala/org/apache/pekko/actor/testkit/typed/scaladsl/TestInbox.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package org.apache.pekko.actor.testkit.typed.scaladsl
+
+import java.util.concurrent.ThreadLocalRandom
+
+import scala.collection.immutable
+
+import org.apache.pekko
+import pekko.actor.{ Address, RootActorPath }
+import pekko.actor.testkit.typed.internal.TestInboxImpl
+import pekko.actor.typed.ActorRef
+import pekko.annotation.{ ApiMayChange, DoNotInherit }
+
+@ApiMayChange
+object TestInbox {
+  def apply[T](name: String = "inbox"): TestInbox[T] = {
+    val uid = ThreadLocalRandom.current().nextInt()
+    new TestInboxImpl((address / name).withUid(uid))
+  }
+
+  private[pekko] val address = RootActorPath(Address("akka.actor.typed.inbox", "anonymous"))
```

Comment:
Does this need to be changed?

Reply:
From what I have seen so far no, there are still some tests that need to be fixed but the actor address doesn't seem to effect anything.